specs/core/0_beacon-chain.md

```diff
@@ -1125,7 +1125,8 @@ A valid block with slot `INITIAL_SLOT_NUMBER` (a "genesis block") has the follow
 `STARTUP_STATE_ROOT` (in the above "genesis block") is generated from the `get_initial_beacon_state` function below. When enough full deposits have been made to the deposit contract and the `ChainStart` log has been emitted, `get_initial_beacon_state` will execute to compute the `hash_tree_root` of `BeaconState`.
 
 ```python
-def get_initial_beacon_state(initial_validator_deposits: List[Deposit],
+def get_initial_beacon_state(initial_validator_registry: List[ValidatorRecord],
+                             initial_validator_deposits: List[Deposit],
```

Comment:
We don't need `initial_validator_registry` as an input. The initial registry is built by processing deposits into the initial state.